### PR TITLE
(BOLT-282) Improve out-of-the-box node configs

### PR DESCRIPTION
**Pull request by MikaelSmith**
Starts with 3 Linux nodes and a Windows node by default. Includes several minor fixes.

### CODE REVIEW

**Comment by nicklewis** [comment]
Section 6 uses `--run_as` when it should be `--run-as`. That wasn't changed as part of this PR, so I can't leave the comment directly on that line.

File: 6-downloading-and-running-existing-tasks/README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ PARAMETERS:
 Let's quickly check on the status of a specific package using `bolt`:
 
 ```
-$ bolt task run package action=status name=bash --nodes node1
```
Comment (nicklewis) [inline 6-downloading-and-running-existing-tasks/README.md]: This fails on `node2` and `node3`, because earlier we only installed puppet on `node1`.

**Comment by MikaelSmith** [comment]
Updated for issues in lab 6.

**Comment by nicklewis** [comment]
Sections 7 and 8 should use `nodes=$NODE` rather than `nodes=<node>`.

Section 8 says to write `foobar.py`, but then calls the metadata file `great_metadata.json`. I think it should be `great_metadata.py`, which is the file that already exists in the repo.

Section 8 also needs `--modulepath ./modules` when running `bolt task show exercise8::great_metadata`.

The last bolt invocation in section 8 should say `if we run again without --noop` instead of `with --noop`.

In Section 9, `modules/exercise9/plans/yesorno.pp` incorrectly creates a plan called `exercise8::yesorno`, when it should be `exercise9::yesorno` (8 vs. 9). It also runs a task which has an `8` in it.

The invocation of the plan in section 9 uses `--nodes=$NODE` when it should use `nodes=$NODE`. Also, the output of the plan invocation is wrong (it includes `ExecutionResult(...)` which actually running the plan does not).

**Comment by MikaelSmith** [comment]
Pushed fixes for exercises 7-9.

**Pull request merged**